src/Features/Core/Portable/Snippets/ConvertToLSPSnippetService.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Snippets
+{
+    internal class ConvertToLSPSnippetService : ILanguageService
```

Comment:
doesn't seem to be an alnguage service (nothing specific that a lanugage needs to do).  this feels more like a static helper method you could put anywhere appropriate.